code/drasil-example/Drasil/GamePhysics/Unitals.hs

```diff
@@ -171,23 +171,23 @@ posCM = ucs "p_CM" (nounPhraseSP "Center of Mass")
   "FIXME: Define this or remove the need for definitions" 
   (sub (eqSymb QP.position) lCMass) Real metre
 
-massI = ucs' (dccWDS "m_j" (compoundPhrase' (QPP.mass ^. term)
+massj = ucs' (dccWDS "m_j" (compoundPhrase' (QPP.mass ^. term)
                 (cn "of the j-th particle")) (phrase QPP.mass)) 
                 (sub (eqSymb QPP.mass) lJ) Real kilogram
 
-posI = ucs' (dccWDS "p_j" (compoundPhrase' (QP.position ^. term) 
+posj = ucs' (dccWDS "p_j" (compoundPhrase' (QP.position ^. term) 
                (cn "vector of the j-th particle")) (phrase QP.position))
                (sub (eqSymb QP.position) lJ) Real metre
 
-accI = ucs' (dccWDS "accI" (compoundPhrase' (cn "the i-th body's")
+accj = ucs' (dccWDS "accj" (compoundPhrase' (cn "the i-th body's")
```

Comment:
Here `accj` has `i` in its English form!

Reply:
Should I change the English description to "j-th body" in order to match the other variable changes. And does the same go for "F_i"?

Reply:
I also noticed that torque and velocity use the "i-th" body in their description too. Should I change those to "j-th"?

Reply:
They should all consistently use the same variable, for now. Putting a variable name in there is a hack, but one we have to live with right now.  Let's do `j`, as `i` is over-used and could clash with other things.